### DISCUSSION TITLE
yfinance api to fetch the options data

### DISCUSSION THREAD

**Discussion by macieman** [aaguiar10/gflows#7] [General]
Can  yfinance api to fetch the options data be used for downloading the options data ?

**Comment by aaguiar10** [comment]
Hi @macieman,

Yes, you can fetch an options chain from there, though I'm not sure how often Yahoo updates that data during a trading session (edit: it's 15min delayed). For details you can refer to the [yfinance](https://github.com/ranaroussi/yfinance) documentation. G|Flows installs the [yahooquery](https://yahooquery.dpguthrie.com/guide/ticker/options/) package.

**Comment by rdyvptnm** [comment]
Is yahoo finance is free ? I need real time steaming and looking for recommendations even if it is paid Sent from my iPhoneOn Mar 27, 2024, at 11:39 PM, Armani Aguiar ***@***.***> wrote:﻿
Hi @macieman,
Yes, you can fetch an options chain from there, though I'm not sure how often Yahoo updates that data during a trading session. For details you can refer to the yfinance documentation. G|Flows installs the yahooquery package.

**Reply by aaguiar10** [reply]
@rdyvptnm,

Yes, it's free to use. However, I checked their option chains today and found that the data is not in real-time.

**Reply by rdyvptnm** [reply]
do you have any other suggestions for real time streaming ?

**Reply by rdyvptnm** [reply]
also does dash support real time chart updating? your current frame work dont have threads and wondering if that works in real time with out any tweaks? Is there a way we can connect together offline to touch base my ideas with you and see if you can help me?

**Reply by aaguiar10** [reply]
@rdyvptnm ,

Email me at bedrckcode@gmail.com to further discuss that topic.
